src/python/CMakeLists.txt

```diff
@@ -501,6 +504,11 @@ if(PYTHONINTERP_FOUND)
       endif()
       add_gudhi_py_test(test_wasserstein_barycenter)
     endif()
+    if(OT_FOUND)
+      if(TENSORFLOW_FOUND AND EAGERPY_FOUND)
+        add_gudhi_py_test(test_wasserstein_with_tensors)
```

Comment:
If the file is about tests with tensorflow, I would put "tensorflow" in the name. With this name, I thought it had both tensorflow and pytorch tests.

Reply:
Sorry about that but this was my first intention. What would you say if I move `test_wasserstein_distance_grad` from `test_wasserstein_distance.py` in `test_wasserstein_with_tensors.py` ? Like that, if you have only pot, at least wasserstein is tested. If you have tensorflow and torch, test_wasserstein_with_tensors is tested (P.S. : I don't know why eagerpy is required here...)

Reply:
ok

Reply:
So did I on f0beb329